### PR TITLE
#00 AS: initializes app with authentication frontend

### PR DESCRIPTION
**Pull request by AdoshSingh**
Initializes the Next.js project
Created frontend for signup and signin pages.

### CODE REVIEW

File: roadmap-voting-app/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
```
Comment (dray92) [inline roadmap-voting-app/README.md:34]: why is this here?

File: roadmap-voting-app/public/vercel.svg
Comment (dray92) [inline roadmap-voting-app/public/vercel.svg:1]: where is this used?

File: roadmap-voting-app/README.md (hunk 1 above)
Comment (AdoshSingh) [reply roadmap-voting-app/README.md:34]: This is the default README, which was automatically generated upon initialization, has been modified in other branches. If required, I can ensure its removal from this branch as well.

File: roadmap-voting-app/public/vercel.svg
Comment (AdoshSingh) [reply roadmap-voting-app/public/vercel.svg:1]: My bad, this was also generated automatically and is currently not in use. So, I will remove it.

File: roadmap-voting-app/README.md (hunk 1 above)
Comment (AdoshSingh) [reply roadmap-voting-app/README.md]: I have made the necessary changes

File: roadmap-voting-app/public/vercel.svg
Comment (AdoshSingh) [reply roadmap-voting-app/public/vercel.svg:1]: I have made the necessary changes

File: roadmap-voting-app/public/next.svg
Comment (dray92) [inline roadmap-voting-app/public/next.svg:1]: is this used anywhere?

**Comment by dray92** [comment]
please address the minor comment before landing

File: roadmap-voting-app/public/next.svg
Comment (AdoshSingh) [reply roadmap-voting-app/public/next.svg:1]: My bad this was also initialized with the create next app, and currently is not being used. I have removed it. I'll take extra care from next time to remove anything that is not being used.
